packages/demo/public/docs/rateLimitingNullifier.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable */
```

Comment:
It can be removed

Reply:
Eslint rules would force to me miss the example code. Because I have have this condition: 

```
then(proof)  {
   // SOME CODE
}
```

And because of Eslint it would say proof is not used. But I need just leave an open example for the user. So I dont need all the eslint rules for the example code.

Reply:
I think users don't need then and catch examples so you can only disable the unused variable not all the rules.